### PR TITLE
Customer docs

### PR DESCRIPTION
**Pull request by ArtemSyromiatnikov**

### CODE REVIEW

File: Anonymization Proxy/anonymization-proxy.md [hunk 1]
```diff
@@ -0,0 +1,48 @@
+## Overview
+
+Anonymization Proxy detects and anonymizes Personally Identifiable Information (PII) that originates from the use of ACE Knowledge Widgets (e.g., performing a search, chatting with an ACE Knowledge Chat Bot, etc.).
+
```
Comment (ArtemSyromiatnikov) [inline Anonymization Proxy/anonymization-proxy.md]: Let's add a line about the purpose of this feature: namely to make sure PII is processed in Sweden (or rather on Telia own infrastructure) and doesn't reach public cloud.

File: Anonymization Proxy/anonymization-proxy.md [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+Please contact ACE Knowledge support team if you wish to start using Anonymization Proxy. Team will enable the feature for your specified tenant(s).
+
+#### Portals
+
+Portal type of Widgets hosted on ACE Knowledge do not require additional configuration as long as they are **publicly accessible**. Please make sure accces mode is set to `Public access` in Widget's Security Settings:
+
+![](images/security-settings.png)
+
+Then, you can simply open Portal Widget by clicking `Preview` on the Widget and choosing `Yes` to `Preview using Anonymization Proxy` option as per below image:
```
Comment (ArtemSyromiatnikov) [inline Anonymization Proxy/anonymization-proxy.md]: "Then you can open Portal Widget..."
- remove comma
- remove 'simply'

**Review by ArtemSyromiatnikov — COMMENTED** [review]
Let's add a purpose message and streamline the style a little.
(I created the PR, so I can't approve it)

File: Anonymization Proxy/anonymization-proxy.md (hunk 1 above)
Comment (aidas-simkus) [reply Anonymization Proxy/anonymization-proxy.md]: Added

File: Anonymization Proxy/anonymization-proxy.md (hunk 2 above)
Comment (aidas-simkus) [reply Anonymization Proxy/anonymization-proxy.md]: Done

File: Anonymization Proxy/anonymization-proxy.md [hunk 3]
```diff
@@ -1,39 +1,40 @@
...
 
-In order to be able to use Anonymization Proxy, please consider the following:
-  * Anonymization Proxy currently only works with public Widgets
-  * Since anonymization of PII is done outside of ACE Knowledge backend, it means that it is no longer possible to search on PII in Widgets when using Anonymization Proxy
-  * It is not possible to use Anonymization Proxy for v1-v3 Widget implementations
-  * Widgets distributed through Web provisions cannot be configured to use Anonymization Proxy
+Before using Anonymization Proxy, please note the following:
+  * Anonymization Proxy currently only works with public Widgets.
+  * Anonymization Proxy does not allow searching on PII in Widgets.
+  * Anonymization Proxy cannot be used for v1-v3 Widget implementations.
```
Comment (ArtemSyromiatnikov) [inline Anonymization Proxy/anonymization-proxy.md]: I think we could reformulate this as a positive message. Something like "AP is applicable to Widgets v4 and OneWidget. Versions v1-v3 are not supported."

File: Anonymization Proxy/anonymization-proxy.md [hunk 4]
```diff
@@ -1,39 +1,40 @@
...
 
 ![](images/security-settings.png)
 
-Then, you can simply open Portal Widget by clicking `Preview` on the Widget and choosing `Yes` to `Preview using Anonymization Proxy` option as per below image:
+Then open the Portal Widget by clicking `Preview` and choosing `Yes` to `Preview using Anonymization Proxy` option:
 
 ![](images/preview.png)
 
-### Widgets embedded on your website
+### Widgets Embedded on Your website
```
Comment (ArtemSyromiatnikov) [inline Anonymization Proxy/anonymization-proxy.md]: Capitalize `Website` as well

**Review by ArtemSyromiatnikov — COMMENTED** [review]
✔ Approved

**Pull request merged**